Library/Formula/svmlight.rb

```diff
@@ -0,0 +1,16 @@
+class Svmlight < Formula
+  homepage "http://svmlight.joachims.org/"
+  url "http://download.joachims.org/svm_light/current/svm_light.tar.gz"
+  version "6.02"
+  sha1 "3b0f843e6f8f4f168d3e33ab957f22c56ccadd86"
+
+  def install
+    system "make all"
+    bin.install "svm_learn"
+    bin.install "svm_classify"
+  end
+
+  test do
+    system "false"
```

Comment:
See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!